bundles/org.openhab.binding.icalendar/src/main/java/org/openhab/binding/icalendar/internal/ICalendarHandlerFactory.java

```diff
@@ -70,16 +75,10 @@ public boolean supportsThingType(ThingTypeUID thingTypeUID) {
         if (!supportsThingType(thingTypeUID)) {
             return null;
         }
-        try {
-            if (!sharedHttpClient.isStarted()) {
-                sharedHttpClient.start();
-            }
-            return new ICalendarHandler(thing, sharedHttpClient, eventPublisher);
-        } catch (Exception e) {
-            logger.warn("Failed to create handler for thing with uid {}.", thing.getUID());
-            logger.debug("internal exception while creating or preparing handler.", e);
+        if (!sharedHttpClient.isStarted()) {
```

Comment:
You can skip this check, we should make sure that it is started.

Reply:
Together with above.